compiler/rustc_ast_lowering/src/expr.rs

```diff
@@ -275,6 +275,12 @@ impl<'hir> LoweringContext<'_, 'hir> {
                     hir::ExprKind::Ret(e)
                 }
                 ExprKind::Yeet(sub_expr) => self.lower_expr_yeet(e.span, sub_expr.as_deref()),
+                ExprKind::Become(sub_expr) => {
+                    let sub_expr = self.lower_expr(sub_expr);
+
+                    // FIXME(explicit_tail_calls): Use `hir::ExprKind::Become` once we implemented it
+                    hir::ExprKind::Ret(Some(sub_expr))
```

Comment:
Would a `span_bug!` make sense here?

Reply:
I guess? Should I change this?

Reply:
if it's easy for you feel free, but i dont think it's too important since you plan on adding the next part soon